pillar/base/haproxy.sls

```diff
@@ -155,3 +171,12 @@ haproxy:
       extra:
         - timeout client 86400
         - timeout server 86400
+
+    {% for (port, service, ssl) in [(25, "smtp", False), (587, "smtps", True), (465, "submission", True)] %}
+    roundup-{{ service }}:
+      bind: :{{ port }} {% if ssl %} ssl crt /etc/ssl/private/bugs.psf.io.pem {% endif %}
```

Comment:
this cert won't exist on the haproxy host, did you mean to use bugs.python.org.pem?

Reply:
Yep 😭

Reply:
[fd51660](https://github.com/python/psf-salt/pull/393/commits/fd51660a6642ac7bd8030aee06cb8a3d7eae66b5)